cognito-pre-token-generation/src/main/java/no/unit/nva/cognito/PostAuthenticationHandler.java

```diff
@@ -197,4 +216,27 @@ private AttributeType toAttributeType(String name, String value) {
             .map(stringRepresentation)
             .collect(Collectors.joining(COMMA_DELIMITER));
     }
+
+    private boolean userIsBibsysHosted(UserAttributes userAttributes) {
+        return userAttributes.getFeideId().endsWith(BIBSYS_HOST)
```

Comment:
Possibly the case that this is always true and simple check for hostedOrgNumber is not sufficient?